__tests__/lib/fetch.test.js

```diff
@@ -9,6 +9,7 @@ describe('#fetch()', () => {
     // List of all GitHub Actions env variables:
     // https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables
     beforeEach(() => {
+      process.env.GITHUB_ACTION = '__repo-owner_name-of-action-repo';
```

Comment:
I added this to further simulate a GHA environment in our tests since this is the variable that [`ci-detect`](https://github.com/npm/ci-detect) uses.